dll/win32/riched20/txthost.c

```diff
@@ -164,35 +164,35 @@ static ULONG WINAPI ITextHostImpl_Release( ITextHost2 *iface )
     return ref;
 }
 
-DEFINE_THISCALL_WRAPPER(ITextHostImpl_TxGetDC,4)
+//DEFINE_THISCALL_WRAPPER(ITextHostImpl_TxGetDC,4)
```

Comment:
I assume it's missing a forward declaration of the function. In that case I would prefer to have these forward declarations all in a header file that is included from the main header, which would make the diff just one line.

Reply:
Added forward declarations in new file riched20.h to minimize diff. Thanks.